Memory error on import

I'm running the command with download media on what is admittedly a large site and then I get this:

```
07:53:16PM INF media_cache_setup.go:81 > media will be fetched url=https://alper.nl/dingen/wp-content/uploads/2009/10/LangeWapper04.jpg
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x10 pc=0x1026fd6f8]

goroutine 1 [running]:
github.com/ashishb/wp2hugo/src/wp2hugo/internal/mediacache.waitOrStop(0x1032e3660?)
        /home/runner/work/wp2hugo/wp2hugo/src/wp2hugo/internal/mediacache/media_cache_setup.go:30 +0x18
github.com/ashishb/wp2hugo/src/wp2hugo/internal/mediacache.MediaCache.GetReader({{0x14000024240?, 0x14000fa9c20?}}, {0x102bd1e30, 0x10330be80}, {0x14003c7cf00, 0x44})
        /home/runner/work/wp2hugo/wp2hugo/src/wp2hugo/internal/mediacache/media_cache_setup.go:100 +0x3a8
github.com/ashishb/wp2hugo/src/wp2hugo/internal/hugogenerator.downloadMedia({_, _}, {_, _}, {_, _}, {_, _, _}, {{0x10275ad4a, ...}, ...}, ...)
        /home/runner/work/wp2hugo/wp2hugo/src/wp2hugo/internal/hugogenerator/hugo_gen_setup.go:768 +0x3a0
github.com/ashishb/wp2hugo/src/wp2hugo/internal/hugogenerator.Generator.downloadPageMedia({{0x10275ad4a, 0x6}, {0x102bce988, 0x14002403440}, {0x16da6aba7, 0x8}, {{0x14000519308, 0x8}, 0x140023a4ab0, {0x1400052d080, ...}, ...}, ...}, ...)
        /home/runner/work/wp2hugo/wp2hugo/src/wp2hugo/internal/hugogenerator/hugo_gen_setup.go:855 +0x334
github.com/ashishb/wp2hugo/src/wp2hugo/internal/hugogenerator.Generator.writePage({{0x10275ad4a, 0x6}, {0x102bce988, 0x14002403440}, {0x16da6aba7, 0x8}, {{0x14000519308, 0x8}, 0x140023a4ab0, {0x1400052d080, ...}, ...}, ...}, ...)
        /home/runner/work/wp2hugo/wp2hugo/src/wp2hugo/internal/hugogenerator/hugo_gen_setup.go:688 +0x150
github.com/ashishb/wp2hugo/src/wp2hugo/internal/hugogenerator.Generator.writePosts({{0x10275ad4a, 0x6}, {0x102bce988, 0x14002403440}, {0x16da6aba7, 0x8}, {{0x14000519308, 0x8}, 0x140023a4ab0, {0x1400052d080, ...}, ...}, ...}, ...)
        /home/runner/work/wp2hugo/wp2hugo/src/wp2hugo/internal/hugogenerator/hugo_gen_setup.go:585 +0x28c
github.com/ashishb/wp2hugo/src/wp2hugo/internal/hugogenerator.Generator.Generate({{0x10275ad4a, 0x6}, {0x102bce988, 0x14002403440}, {0x16da6aba7, 0x8}, {{0x14000519308, 0x8}, 0x140023a4ab0, {0x1400052d080, ...}, ...}, ...}, ...)
        /home/runner/work/wp2hugo/wp2hugo/src/wp2hugo/internal/hugogenerator/hugo_gen_setup.go:110 +0x140
main.generate({_, _}, {{0x14000519308, 0x8}, 0x140023a4ab0, {0x1400052d080, 0x1b}, 0x1400000e798, {0x14000519558, 0x5}, ...}, ...)
        /home/runner/work/wp2hugo/wp2hugo/src/wp2hugo/cmd/wp2hugo/main.go:80 +0x28c
main.handle({0x102bd1e30, 0x10330be80}, {0x16da6abb9, 0x2a})
        /home/runner/work/wp2hugo/wp2hugo/src/wp2hugo/cmd/wp2hugo/main.go:60 +0xf4
main.main()
        /home/runner/work/wp2hugo/wp2hugo/src/wp2hugo/cmd/wp2hugo/main.go:46 +0xdc
```

I must say it's not often that I see this kind of memory error in Go.

--- a/src/wp2hugo/internal/mediacache/media_cache_setup.go
+++ b/src/wp2hugo/internal/mediacache/media_cache_setup.go
@@ -32,7 +32,7 @@ func waitOrStop(resp *http.Response) (int, bool) {
 	// May want to bump the timeout but since we've not got a valid
 	// response, it's not entirely clear if we even made a request.
 	if resp == nil {
-		return timeout, stop
+		return timeout, false
 	}
 
 	switch resp.StatusCode {

